OpenRA.Game/Server/Server.cs

```diff
@@ -392,7 +392,7 @@ void ValidateClient(Connection newConn, string data)
 						SendOrderTo(newConn, "Message", motd);
 				}
 
-				if (!LobbyInfo.IsSinglePlayer && Map.DefinesUnsafeCustomRules)
+				if (!LobbyInfo.IsSingleHuman && Map.DefinesUnsafeCustomRules)
```

Comment:
Checking the player count here is a bug: it suppresses the message if somebody is joining an empty server that defaults to a map with custom rules.